### PR TITLE
Load order information from RobinhoodOrder objects.

### PR DESCRIPTION
**Pull request by mike-roberts1**

### CODE REVIEW

File: webapp/cached.py [hunk 1]
```diff
@@ -9,7 +9,9 @@ def __init__(self, *args, **kwargs):
 
     def __call__(self, func):
         def inner(*args, **kwargs):
-            param = args[1]
+            param = args[0]
```
Comment (rosscooperman) [inline webapp/cached.py]: does python not have ternaries 😛 ?

File: webapp/stockstream/stockstream_api.py [hunk 2]
```diff
@@ -86,14 +86,14 @@ def get_orders_by_id(order_ids):
         request = "/v1/orders?ids={}".format(",".join(chunk))
         votes = get_api_request(request)
         all_orders += votes
-    orders = sorted(all_orders, key=lambda k: k['timestamp'])
+    orders = sorted(all_orders, key=lambda k: k['created_at'])
```
Comment (rosscooperman) [inline webapp/stockstream/stockstream_api.py:89]: I realize these are tweaks to existing code but this kinda feels like basically the same code 4 times in a row.

File: webapp/cached.py (hunk 1 above)
Comment (mike-roberts1) [reply webapp/cached.py]: Updated- there's a ternary operator using same-line if-else statements. I usually prefer standard if-else, but here I guess it makes sense.

File: webapp/stockstream/stockstream_api.py (hunk 2 above)
Comment (mike-roberts1) [reply webapp/stockstream/stockstream_api.py:89]: Yeah kinda ugly- though I can't think of a cleaner way to do this. The website needs orders to be sorted by their timestamp but the API service does not guarantee it- I don't think it makes sense to do it there either. I do play on refactoring this python stockstream client into an object-oriented pattern and can fix it up then.

**Review by rosscooperman — APPROVED** [review]

**Pull request merged**